stubs/QueryInterface.stub

```diff
@@ -6,12 +6,6 @@ namespace TYPO3\CMS\Extbase\Persistence;
  */
 interface QueryInterface
 {
-    /**
-     * @param bool $returnRawQueryResult
-     * @return \TYPO3\CMS\Extbase\Persistence\QueryResultInterface<ModelType>|array<string, mixed>
```

Comment:
We have now updated the second type to be a `list`: https://github.com/sascha-egerer/phpstan-typo3/commit/b56227cda281d6bc90ebd54526d33d7f39f034e6

Do we then need changes to the `DynamicReturnTypeExtension` in this PR?